third_party/dependency_analyzer/src/main/io/bazel/rulesscala/dependencyanalyzer/DependencyAnalyzer.scala

```diff
@@ -77,6 +80,19 @@ class DependencyAnalyzer(val global: Global) extends Plugin {
 
   }
 
+  private def getJarLabel(jarPath: String): Option[String] = {
+    // First check the jar's manifest for a stamped label
+    Try(new JarFile(jarPath)).toOption.flatMap { jf ⇒
+      try {
```

Comment:
can we change to:
```scala
try {
  val jf = new JarFile(jarPath)
  ...
```
rather than `Try().toOption` since we have a try following anyway? Seems a bit clearer